web/src/localization/locale/sv.ftl

```diff
@@ -8,6 +8,7 @@ stacklet--add = Lägg till Stacklet
 stacklet--product = Produkt
 stacklet--namespace = Namnrymd
 stacklet--name = Namn
+stacklet--endpoints = Endpoints
```

Comment:
Let's check with @nightkr if this is the correct translation.

Reply:
Yeah, I did plan to tag here but forgot. It may also be "Slutpunkter", but "Endpoint" seems like a safer bet to me xD

Reply:
Should we block on this or merge anyway?

Reply:
I think I would merge it anyway, especially as we have no timeline on when she is back. Happy to change it in a follow PR once we have feedback

Reply:
Okay. Marking as resolved, let's move forward then.